homeassistant/components/asuswrt/device_tracker.py

```diff
@@ -34,7 +34,9 @@ def update_router():
 
 
 @callback
-def add_entities(router, async_add_entities, tracked):
+def add_entities(
+    router: AsusWrtRouter, async_add_entities: AddEntitiesCallback, tracked: set
```

Comment:
Can we type the contents of the set?

Reply:
How this can be done?

Reply:
The same way as for a list.
```py
set[ITEM_TYPE]
```

Reply:
Didn't know it was possible for a Set ... continuous learning 🎉